v2/tools/mangle-test-json/main.go

```diff
@@ -34,6 +35,7 @@ type TestRun struct {
 
 func main() {
 	for _, testOutputFile := range os.Args[1:] {
+		log.Printf("Parsing  %s\n\n", testOutputFile)
```

Comment:
Does this go to `stderr?`

Reply:
Sounds like yes.

Reply:
Yes - appears in the action log, not in the markdown file:

```
[produce-markdown-summary] 2022/05/15 20:49:21 Parsing  /workspace/reports/controller-integration-tests.json
[produce-markdown-summary] 
[produce-markdown-summary] 2022/05/15 20:49:21 Complete. 
```